tests/db_functions/text/test_md5.py

```diff
@@ -0,0 +1,34 @@
+from django.db.models import CharField
+from django.db.models.functions import MD5
+from django.test import TestCase
+from django.test.utils import register_lookup
+
+from ..models import Author
+
+
+class MD5Tests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        Author.objects.bulk_create([
+            Author(name='John Smith', alias='a'),
+            Author(name='Jordan Élena', alias='b'),
+            Author(name='皇帝', alias='c'),
+            Author(name='', alias='d'),
+        ])
+
+    def test_basic(self):
+        authors = Author.objects.annotate(md5_name=MD5('name'))
+        self.assertQuerysetEqual(
+            authors.order_by('alias'), [
```

Comment:
Could drop the `alias` in `setUpTestData` and `order_by('pk')` or simply use `assertCountEqual` since ordering isn't that important.